docs/AppBuildAndDeploy.md

```diff
@@ -134,14 +137,16 @@ extends:
     deployFromFeature: ${{ parameters.deployFromFeature }}  #Mandatory: True/False(default)  parameter used to deploy feature branch to dev environment.
     deployConfigOnly: ${{ parameters.deployConfigOnly }}    #Mandatory: True/False(default)  parameter used to deploy app config to various environments.
     privateAgentName: 'DEFRA-ubuntu2204'                    #Optional:  Name of the private build agent. default will use Azure hosted linux agent.
-    packageFeedName: 'artifact-feed'                        #Mandatory: Name of the Azure Devops Artifacts package feed. Used by .Net and NodeJs build.
+    packageFeedName: 'artifact-feed'                        #Optional: Name of the Azure Devops Artifacts package feed. Used by .Net and NodeJs build.
     appBuildConfig: ${{ parameters.appBuildConfig }}        #Mandatory: Object which contains configration used for building the application. Such as appFrameworkType, defaultBranch, frameworkVersion, projectPath, manifestPath, imageRepoName
+      githubreleaseconnection: 'github connection'          #Mandatory: Should be a OAuth or PAT. Used to tag and create github release
```

Comment:
Can be defaulted to 'DEFRA' as we'll scaffold projects using that?

Reply:
but documentation is right in general

Reply:
DEFRA connection is not OAuth it is based on "azure pipelines app". Hence had to create new connection.

Reply:
@ppotturi  are we sharing the same/correct connection for ADP Portal API?

Reply:
I didn't get it. Can we have a call?

Reply:
Added task to the backlog: https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/306312